src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -125,11 +126,11 @@ class AdditionalDetailsStep extends React.Component {
         }
 
         if (!ValidationUtils.isValidPastDate(values[INPUT_IDS.DOB])) {
-            errors[INPUT_IDS.DOB] = this.props.translate(this.errorTranslationKeys.dob);
+            errors = ErrorUtils.addErrorMessage(errors, INPUT_IDS.DOB, this.props.translate(this.errorTranslationKeys.dob));
```

Comment:
ErrorUtils.addErrorMessage copies reference and mutates error object, which is not needed to assign a value.